### ISSUE TITLE
Getting perfect Lighthouse score

### ISSUE THREAD

**Issue by blazeu** [coala/gci-leaders#55]
>Lighthouse analyzes web apps and web pages, collecting modern performance metrics and insights on developer best practices.

https://github.com/GoogleChrome/lighthouse

This is our current Lighthouse audit result: https://u.teknik.io/4SV0f.html

This task is for increasing our Lighthouse score, multiple students can take this task.

For now, don't worry about service worker / offline support, I think we will have another task for that.

Your build must be rebased and have at least one more audit passed than master.

**Comment by jayvdb** [comment]
> u.teknik.io/4SV0f.html

connection failure.  They fail too?

**Comment by Mixih** [comment]
ya, they've misconfigured their ssl it seems

**Comment by myfrom** [comment]
Can I take the Best Practises and Accessibility? I could also try my best at Performance.

For [GCI task](https://codein.withgoogle.com/dashboard/task-instances/5311708543320064/)

**Comment by blazeu** [comment]
@myfrom Sure.

**Comment by myfrom** [comment]
Just a question: Do you have any favicon? It's not really related to the task but I saw you're setting og:image to GCI logo but leaving favicon blank and I think we could set GCI logo as favicon too.

**Comment by blazeu** [comment]
Yes, of course.

**Comment by myfrom** [comment]
I have one more question: I have created favicon.ico, but I'm not sure how to tell the build script to put it into the output, how should I do that?

**Comment by blazeu** [comment]
Yes, while you're at it, modify it so that it copies the content of `static` into `out/` not `out/static` so that we're able to put something in the root folder of `out` such as favicon or manifest.

Don't forget to split your commit accordingly.
